From 1351a6b8f09ab2f4a7469d6e2be874b56a31d3c4 Mon Sep 17 00:00:00 2001
From: Brian Dolbec <dolsen@gentoo.org>
Date: Thu, 28 Aug 2014 20:56:22 -0700
Subject: [PATCH 03/14] equery has: Add CFLAGS, CXXFLAGS, LDFLAGS to strip the
 leading '-'

Strip the leading '-' from values found in these files.
---
 pym/gentoolkit/equery/has.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pym/gentoolkit/equery/has.py b/pym/gentoolkit/equery/has.py
index 15c60b9..180f7f0 100644
--- a/pym/gentoolkit/equery/has.py
+++ b/pym/gentoolkit/equery/has.py
@@ -74,7 +74,7 @@ def query_in_env(query, env_var, pkg):
 	"""Check if the query is in the pkg's environment."""
 
 	try:
-		if env_var in ("USE", "IUSE"):
+		if env_var in ("USE", "IUSE", "CFLAGS", "CXXFLAGS", "LDFLAGS"):
 			results = set(
 				[x.lstrip("+-") for x in pkg.environment(env_var).split()]
 			)
-- 
2.1.3

